detox.config.js

```diff
@@ -0,0 +1,77 @@
+/** @type {Detox.DetoxConfig} */
+module.exports = {
+  logger: {
+    level: process.env.CI ? 'debug' : undefined,
+  },
+  testRunner: {
+    args: {
+      config: 'e2e/jest.config.js',
+      maxWorkers: 1,
+    },
+  },
+  artifacts: {
+    plugins: {
+      log: process.env.CI ? 'failing' : undefined,
+      screenshot: process.env.CI ? 'failing' : undefined,
+    },
+  },
+  apps: {
+    'ios.debug': {
+      type: 'ios.app',
+      binaryPath: 'ios/build/Build/Products/Debug-iphonesimulator/BareExpoDetox.app',
+      build: './scripts/build-detox-ios.sh Debug',
```

Comment:
This script seems to be missing in this repository. Did you forget to commit it? 🤔

Reply:
Not sure this script is needed, will check this

Reply:
```suggestion
```